sdk/keyvault/keyvault-keys/src/index.ts

```diff
@@ -408,7 +421,9 @@ export class KeyClient {
 
       return this.getKeyFromKeyBundle(response);
     } else {
-      const response = await this.client.createKey(this.vaultUrl, name, "EC", options);
+      const response = await this.client.createKey(this.vaultUrl, name, {
+        kty: "EC"
+      });
```

Comment:
I removed "options" since by this point options is undefined. See the if conditional above in the code (you should be able to see the `} else  {` part from here).